src/client/utils/dom.js

```diff
@@ -363,11 +363,12 @@ export function isIframeWithoutSrc (iframe) {
     var parentWindowLocation       = parsedParentWindowLocation ? parsedParentWindowLocation.destUrl : parentWindowWithSrc.location.toString();
 
     if (iframeDocumentLocationHaveSupportedProtocol) {
-        // NOTE: After document.open is called for a same-domain iframe,
-        // the iframe window location becomes equal to the location of the parent window with a src.
-        var parsedIframeSrcLocation = urlUtils.parseUrl(iframeSrcLocation);
+        // NOTE: In IE, after document.open is called for a same-domain iframe or iframe with a javascript src,
+        // the iframe window location becomes equal to the location of the parent window with the src.
```

Comment:
These article issues... while I agreed not to insert missing articles, here there is a redundant article, which looks weirder than missing ones. So fix please.
```
        // NOTE: In IE, after document.open is called for a same-domain iframe or an iframe with a javascript src,
        // the iframe window location becomes equal to the location of the parent window with src.
```